tests/ks_initial/test.py

```diff
@@ -0,0 +1,12 @@
+from pytest import fixture
+import numpy as np
+from pathlib import Path
+
+
+@fixture
```

Comment:
use `conftest.py` file,
see https://towardsdatascience.com/testing-best-practices-for-machine-learning-libraries-41b7d0362c95
also fragment about `conftest.py` in 👍 https://docs.pytest.org/en/7.1.x/reference/fixtures.html?highlight=conftest%20py